sdk/storage/Azure.Storage.Blobs.Cryptography/src/Models/EncryptionAlgorithm.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Storage.Blobs.Specialized.Cryptography.Models
+{
+    /// <summary>
+    /// Specifies the encryption algorithm used to encrypt a resource.
+    /// </summary>
+    public enum ClientsideEncryptionAlgorithm
+    {
+#pragma warning disable CA1707 // Identifiers should not contain underscores; supressed for serialization dependencies
+        /// <summary>
+        /// AES-CBC using a 256 bit key.
+        /// </summary>
+        AES_CBC_256
```

Comment:
Can we make this look more like the naming in KeyVault?   https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/keyvault/Microsoft.Azure.KeyVault.Cryptography/src/Algorithms/Aes128CbcHmacSha256.cs#L10

Reply:
I'm not sure which part(s) of that you're going for.

Reply:
Using the name `AesCbc256`.